apiary.apib

```diff
@@ -6,6 +6,31 @@ OpenGrok RESTful API documentation. The following endpoints are accessible under
 
 Besides `/suggester` and `/search` endpoints, everything is accessible from `localhost` only.
 
+## Annotation [/annotation{?path}]
+
+### Get annotation for a file [GET]
+
++ Parameters
+  + path (string) - path of file, relative to source root
+
++ Response 200 (application/json)
+  + Body
+
+            [
+              {
+                "number": 1,
```

Comment:
what is the number?

Reply:
just entry number, I guess that's not necessary.

Reply:
you can include it but give it better name :-D

Reply:
it would make sense to have it if pagination was supported.